Web/Pagination.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(init);
+
+function init() {
+	$("#pg1").on('click', displayTable);
+	$("#pg2").on('click', displayTable1);
+	$("#pg3").on('click', displayTable2);
+	$("#pg4").on('click', displayTable3);
+	$("#pg5").on('click', displayTable4);
+}
+
+function displayTable() {
+	var query = "<table style='border:1px solid black; width:300px;'><tr><th>Movie Name</th><th>Movie Review</th></tr><tr><td>Jab We Met</td><td>4</td></tr><tr><td>3 Idiots</td><td>5</td></tr><tr><td>Shanghai</td><td>1</td></tr></table>";
+	document.getElementById("details").innerHTML = query;
+}
+
+function displayTable1() {
```

Comment:
Better to use single method and pass the parameters like Movie Name, Movie Review etc. Think about hundred movies list or so on.